packages/amplify-nodejs-function-template-provider/resources/lambda/crud/package.json.ejs

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "<%= props.functionName.toLowerCase() %>",
+"name": "<%= props.functionName.toLowerCase() %>",
   "version": "1.0.0",
   "description": "Lambda function generated by Amplify",
   "main": "index.js",
   "license": "Apache-2.0",
   "dependencies": {
-    "aws-serverless-express": "^3.3.5",
-    "body-parser": "^1.19.1",
-    "express": "^4.17.2"
+  "aws-serverless-express": "^3.3.5",
+  "body-parser": "^1.19.1",
+  "express": "^4.17.2"
   },
   "devDependencies": {
-    "@types/aws-lambda": "^8.10.92",
-    "aws-sdk": "^2.1113.0"
+  "@types/aws-lambda": "^8.10.92",
+  "aws-sdk": "^2.1169.0"
```

Comment:
This is an ejs template for functions added to an Amplify project. I think changing this is out of scope for this PR.